src/main/kotlin/no/nav/familie/ef/sak/service/JournalføringService.kt

```diff
@@ -41,14 +41,14 @@ class JournalføringService(private val journalpostClient: JournalpostClient,
         val behandling: Behandling = hentBehandling(journalføringRequest)
         val journalpost = hentJournalpost(journalpostId)
 
+        settSøknadPåBehandling(journalpostId, behandling.fagsakId, behandling.id)
```

Comment:
Ser ut som denne feiler innimellom pga autorisasjon i preprod, mens oppdatering/ferdigstilling av journalpost og oppgave går finfint. Greit å feile her før man får ferdigstilt... Mulig vi burde hatt en "rollback"-håndtering av ferdigstilling av oppgave/journalpost også?

Reply:
Hvor feiler det pga authorisasjon? I integrasjoner eller når integrasjoner kaller videre? Og hvorfor feiler den pga autorisasjon? 
For å unngå att ett senere kall feiler så kan det nok være å splitte opp dette i en task

Reply:
Fordi du ikke får lov å hente søknaden som innlogget saksbehandler - pga tilgangskontroll i SAF, men du får oppdatere journalføring, ferdigstille journalføring og ferdigstille oppgaven 😱 SAF er streng på dette med riktig enhet og behandlingstema, men det er ikke dokarkiv og oppgave. Så vidt jeg har forstått skal dette løsnes på og alle enheter skal kunne jobbe på tvers (minus Vikafossen/kode6) 

```
hentDokument hentet ikke dokument. journalpostId=453640497, dokumentInfoId=454021321, variantFormat=ORIGINAL. Tilgang ble avvist av grunn: Avvist av SAF tilgangskontroll: Tilgang til ressurs ble avvist. Saksbehandler eller system har ikke tilgang til tema ressurset tilhører eller geografisk område.	
```